distributions/standalone/src/main/resources/conf/wrapper.conf

```diff
@@ -43,9 +43,6 @@ wrapper.java.additional.15=-Dorg.quartz.scheduler.skipUpdateCheck=true
 # (see http://wrapper.tanukisoftware.org/doc/english/prop-java-additional-n.html)
 # *** IMPORTANT ***
 
-# Stack traces are truncated in the logs by default
-#wrapper.java.additional.<n>=-Dmule.verbose.exceptions=true
```

Comment:
Similar to previous comment, delete only the properties that are configurable from the XML